bioprojects/PRJNA525405/metadata/create_prussin_metadata.py

```diff
@@ -0,0 +1,46 @@
+import re
+import csv
+from collections import defaultdict
+sra_run_path = "sraruninfo.csv"
+disease_tsv_path = "biosample.txt" 
```

Comment:
Document where these files came from and how you got them.

Reply:
done